ci/docker/image-builder/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+import org.jetbrains.kotlin.gradle.dsl.KotlinJvmProjectExtension
+
+plugins {
+    id("org.jetbrains.kotlin.jvm") version "1.6.10"
+    id("nebula.integtest") version "9.5.2"
+    application
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
+    implementation("org.jetbrains.kotlinx:kotlinx-cli:0.3.3")
+
+    testImplementation("org.junit.jupiter:junit-jupiter:5.8.0")
+
+    integTestImplementation("org.junit.jupiter:junit-jupiter:5.8.0")
+}
```

Comment:
As minimum external dependencies as possible to have fewer points of change over time.